lib/ExecutionControlEpic/PlanConfigurationFeature/Reducers/PlanConfigs.js

```diff
@@ -31,7 +31,14 @@ export default function(
         return state.update(
           action.payload.tool.id,
           (plansForTool: Map<string, PlanConfig>) =>
-            plansForTool.filter((plan: PlanConfig) => !plan.autoGenerated),
+            plansForTool.filter(
+              (plan: PlanConfig) =>
+                !(
+                  plan.autoGenerated &&
+                  // TODO - better handle grouping plans by package
```

Comment:
Instead of writing a TODO comment that we'll forget soon, it'd be better to make a card on our internal trello.